src/Microsoft.Crank.RegressionBot/Program.cs

```diff
@@ -922,7 +924,7 @@ private static async IAsyncEnumerable<Regression> FindRegression(Source source)
                             var regression = new Regression
                             {
                                 PreviousResult = results[i - 1],
-                                CurrentResult = results[i],
+                                CurrentResult = results[i - 1],
```

Comment:
It's not clear to me why the current and previous results are identical here.

Reply:
This model is made for detecting regressions, with Previous/Current, but here I am using the same bot/model to report the fact that the benchmark stopped running. So we can only provide a single result, which is the last one. Another option might be to set PreviousResult to null, it's more logical, but at the same time this could lead to null refs if a template is not expecting that.

Reply:
Add a comment explaining why they have the same result